lab_1/main.py

```diff
@@ -12,7 +11,20 @@ def tokenize(text: str) -> list:
     e.g. text = 'The weather is sunny, the man is happy.'
     --> ['the', 'weather', 'is', 'sunny', 'the', 'man', 'is', 'happy']
     """
-    pass
+
+    sentences = ''
+    text1 = text.lower()
+    slices = list(text1)
+    l = ["”", "“", "^", "|", "}", "{", "]", "[", "&", "%", "№", "#", "`", "`", "'", "<", ">", ".", ",", "!", "?", ";", ":", "*", "’", "$", "@", "*", "/", "-", "_", "+", "=", "(", ")", "1", "2", "3",
+             "4", "5", "6", "7", "8", "9", "0"]
+    for y in slices:
+        for y in l:
```

Comment:
Please, do not use same variable name inside loops,
You have two `y` variables - Python can be confused and act unexpectedly.